openmetadata-ui/src/main/resources/ui/src/pages/AddQueryPage/AddQueryPage.component.tsx

```diff
@@ -102,7 +115,11 @@ const AddQueryPage = () => {
         SearchIndex.TABLE
       );
       const options = data.hits.hits.map((value) => ({
-        label: getEntityName(value._source),
+        label: getLabelValue(
+          value._source.displayName || '',
+          value._source.name,
+          value._source.fullyQualifiedName || ''
+        ),
```

Comment:
since we are extracting the values from the same object we can pass the whole object and extracting the values which we needed ?

Reply:
Passed the whole object and extracted values from it.